lib/Common/Memory/Recycler.inl

```diff
@@ -543,7 +543,10 @@ Recycler::NotifyFree(T * heapBlock)
         this->isForceSweeping = true;
         heapBlock->isForceSweeping = true;
 #endif
-        heapBlock->template SweepObjects<SweepMode_InThread>(this);
+#ifdef RECYCLER_TRACE
+        this->PrintBlockStatus(nullptr, heapBlock, _u("[**34**] calling SweepObjects during NotifyFree."));
```

Comment:
What does 34 here mean?

Reply:
:) It doesn't have any particular significance, apart from the fact that the lower numbers mean early stages of the sweep and higher numbers mean later stages. It only tells me where in code this step happened. Across multiple GCs as heap blocks go through a journey I found this useful in finding out if things are making progress the way we expect. This kind of tracing will be enabled only in verbose mode to diagnose/understand how certain type of heap blocks are processed during sweep.

---
In reply to: [148167948](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148167948) [](ancestors = 148167948)